build.assets/genproto.sh

```diff
@@ -25,15 +25,14 @@ main() {
     --path=api/proto/teleport/loginrule/ \
     --path=api/proto/teleport/proxy/ \
     --path=proto/teleport/lib/multiplexer/
-  buf generate --template=lib/prehog/buf.gen.yaml lib/prehog/proto
-
-  # Generate lib/teleterm & JS protos.
-  # TODO(ravicious): Refactor generating JS protos to follow the approach from above, that is have a
-  # separate call to generate Go protos and another for JS protos instead of having
-  # teleterm-specific buf.gen.yaml files.
-  # https://github.com/gravitational/teleport/pull/19774#discussion_r1061524458
-	buf generate --template=lib/prehog/buf-teleterm.gen.yaml lib/prehog/proto
-	buf generate --template=lib/teleterm/buf.gen.yaml lib/teleterm/api/proto
+  buf generate --template=buf-go.gen.yaml lib/teleterm/api/proto
```

Comment:
Could you use --path and append this to the command above?

Reply:
This won't work as `lib/teleterm/api/proto` is defined as a workspace directory.

> Failure: path "lib/teleterm/api/proto/" is equal to workspace directory "lib/teleterm/api/proto" defined in "buf.work.yaml"

Reply:
I'm wondering if we need workspaces at all, let me check if it'd be possible to do without them after moving everything under the proto directory.